bedrock/firefox/templates/firefox/concerts.html

```diff
@@ -54,6 +54,8 @@
             <a class="video-play" href="http://firefoxconcerts.livenation.com" rel="external" title="Watch the livestream">
               <img src="{{ static('img/firefox/concerts/video-poster-phosphorescent.jpg') }}" alt="Phosphorescent (Matthew Houck) and his six-piece band perform on stage before a crowd. The lighting is tinted purple and the band is backed by a large, illuminated sign reading ‘C’est La Vie’, the title of their recent album.">
             </a>
+          {% elif switch('firefox-concert-one-over') %}
```

Comment:
We're trying to move to underscores for switches. Maybe match `firefox_concert_series`?